modules/network/src/integrationTest/java/org/apache/ignite/network/scalecube/ITScaleCubeNetworkMessagingTest.java

```diff
@@ -48,6 +48,11 @@
 
 /** */
 class ITScaleCubeNetworkMessagingTest {
+    /**
+     * Test cluster.
+     * <p>
```

Comment:
`p` should have a closing tag

Reply:
why?

Reply:
Hm, I always thought that p requires a closing tag or at least it should be self-closing. Ok then)